.github/workflows/database.yml

```diff
@@ -32,15 +32,13 @@ jobs:
     # Don't run on private repo unless it is a PR.
     if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
     runs-on: macos-latest
-    strategy:
-      matrix:
-        target: [iOS, tvOS, macOS]
     steps:
     - uses: actions/checkout@v2
     - name: Setup Bundler
       run: scripts/setup_bundler.sh
     - name: IntegrationTest
-      run: scripts/third_party/travis/retry.sh scripts/build.sh Database ${{ matrix.target }} integration
+      # Only iOS to mitigate flakes.
```

Comment:
I wonder may the flakes occur due to concurrent tests execution (modifying the same database keys from tests fro different platforms simultaneously)? If so, we probably can fix it by modifying the tests, but it's probably out of scope of the PR.

Reply:
Yep, I wondered the same thing - but we've also disabled other tests in this suite because of flakiness from a single run. 

In any case until we invest in improving the robustness of these tests, its not worth the cost of running the integration tests for non-iOS platforms.